pkg/minishift/network/utils.go

```diff
@@ -104,3 +114,23 @@ func HasNameserverConfiguredLocally(nameserver string) (bool, error) {
 
 	return strings.Contains(string(file), nameserver), nil
 }
+
+// AllowInsecureCertificatesOnLocalConnections will not verify certificates for TLS connections
+func OverrideInsecureSkipVerifyForLocalConnections(insecureSkipVerify bool) error {
+	http.DefaultTransport.(*http.Transport).TLSClientConfig = &tls.Config{InsecureSkipVerify: insecureSkipVerify}
+	// We need to force a connection, else this will not work
+	_, err := http.Get("http://localhost")
+	return err
+}
```

Comment:
Does this mean user should not see an `not secure` warning for openshift web console?

Reply:
This has nothing to do with certificates served by a webserver... so no, this does NOT solve this self-signed certificate issue